pennylane/transforms/diagonalize_measurements.py

```diff
@@ -60,6 +60,14 @@ def diagonalize_measurements(tape, supported_base_obs=_default_supported_obs, to
         circuit measurements, the :func:`split_non_commuting <pennylane.transforms.split_non_commuting>` transform
         can be applied.
 
+    .. note::
```

Comment:
It looks kind of funny that there's a note right above this one, and then this second note with two separate things inside it. Should we make them either one large note, or three individual notes?

Reply:
Updated it to one large note. Let me know if you have a different preference here.